tests/test_module_doc.py

```diff
@@ -35,6 +35,7 @@
 # Ignored items will not have their default values checked or be included for
 # alphabetical order purposes
 IGNORE_ITEM = [
+    ('window_list', 'format_window'),  # line too long for docstring parsing
```

Comment:
This would break the full customization (eg different icons and/or names)... as well as controlling how windows should appear.